src/processor/Package.ts

```diff
@@ -228,4 +229,59 @@ export class Package {
       pullAt(sd.rules, rulesToRemove);
     });
   }
+
+  private removeDefaultSlicingRules(): void {
+    [...this.profiles, ...this.extensions].forEach(sd => {
+      const rulesToMaybeRemove: number[] = [];
+      sd.rules.forEach((rule, i, allRules) => {
+        if (
+          rule instanceof ExportableCaretValueRule &&
+          (rule.path.endsWith('extension') || rule.path.endsWith('modifierExtension'))
+        ) {
+          const hasSlicingDiscriminatorTypeRule =
+            rule.caretPath === 'slicing.discriminator[0].type' &&
+            rule.value instanceof FshCode &&
+            rule.value.code === 'value';
+          const hasSlicingDiscriminatorPathRule =
+            rule.caretPath === 'slicing.discriminator[0].path' && rule.value === 'url';
+          const hasSlicingOrderedRule =
+            rule.caretPath === 'slicing.ordered' && rule.value === false;
+          const hasSlicingRulesRule =
+            rule.caretPath === 'slicing.rules' &&
+            rule.value instanceof FshCode &&
+            rule.value.code === 'open';
```

Comment:
The above four work, but @mint-thompson had recommended a slightly different approach in a PR I recently put up.  They suggested creating the default rules and then comparing using `isEqual`.  Here's the result: https://github.com/FHIR/GoFSH/blob/master/src/processor/Package.ts#L172-L179

If you were to consider that, then you would define:
```ts
const DEFAULT_SLICING_DISCRIMINATOR_TYPE_RULE = new ExportableCaretValueRule(rule.path);
DEFAULT_SLICING_DISCRIMINATOR_TYPE_RULE.caretPath = 'slicing.discriminator[0].type';
DEFAULT_SLICING_DISCRIMINATOR_TYPE_RULE.value = new FshCode('value');
```
Then in your `if` statement around line 265, you would have:
```ts
isEqual(rule, DEFAULT_SLICING_DISCRIMINATOR_TYPE_RULE)
```

I'm not sure if it's better in this case or not, but it's something to consider.

Reply:
Ah yes, I'll update to use this approach. I had started off with this, but had changed in when refactoring some things, but I think it makes the rules a lot easier to understand, so I'll update!